dist/cms.js

```diff
@@ -5,9 +5,15 @@ function usesBlockTheme() {
 }
 
 const wordpress = {
-  block_theme: usesBlockTheme()
+  block_theme: usesBlockTheme(),
+  has_embed_block: hasWordPressEmbedBlock()
 };
 
 return {
   wordpress
 };
+
+// Detects if a WordPress embed block is on the page
+function hasWordPressEmbedBlock() {
+  return !!document.querySelector('div.wp-block-embed');
```

Comment:
One offline question about figure vs div, otherwise my only other comment would be to hoist this function higher in the file after `usesBlockTheme`